src/main/scala/monocle/Optic.scala

```diff
@@ -176,9 +182,9 @@ extension [ThisCan, S, T, A, B] (self: POptic[ThisCan, S, T, Option[A], Option[B
   def some: POptic[ThisCan | GetOption, S, T, A, B] = 
     self.andThen(std.option.pSome)
 
-// extension [S, T, A, B] (self: POptic[GetMany, S, T, A, B])
-//   def to[C](f: A => C): POptic[GetMany, S, Any, C, Nothing] = 
-//     self.andThen(Optic.thatCan.get(f))
+extension [S, T, A, B] (self: POptic[GetMany, S, T, A, B])
```

Comment:
what if the current optic is a `Lens` or `Optional`? In monocle, `to` on `Lens` returns a `Getter`, not a `Fold`
https://github.com/optics-dev/Monocle/blob/master/core/shared/src/main/scala/monocle/Lens.scala#L132